Lib/fontTools/cffLib/__init__.py

```diff
@@ -1027,9 +1027,22 @@ def buildConverters(table):
 	return d
 
 
-class SimpleConverter(object):
+class BaseConverter(object):
```

Comment:
SimpleCoverter is really the BaseConverter, as it's only used as base class and never used as such. so you can define read and _read on SimpleConverter itself.